checkseum.opam

```
@@ -41,4 +38,5 @@ depopts: [
 conflicts: [
   "mirage-xen-posix" {< "3.1.0"}
   "ocaml-freestanding" {< "0.4.1"}
+  "mirage" {< "4.0.0"}
```

```suggestion
  "mirage-runtime" {< "4.0.0"}
```
please conflict with `mirage-runtime`, not mirage, since the runtime is what a unikernel depends on, not the tool mirage itself. thanks!